heartbeat/sybaseASE

```diff
@@ -0,0 +1,895 @@
+. ${OCF_FUNCTIONS_DIR}/ocf-shellfuncs
+
+#######################################################################
+
+# Default timeouts when we aren't using the rgmanager wrapper
+if ! ocf_is_true "$OCF_RESKEY_is_rgmanager_wrapper"; then
+	if [ -z "$OCF_RESKEY_CRM_meta_timeout" ]; then
+		case $1 in
+			start|stop) OCF_RESKEY_CRM_meta_timeout=300000 ;;
+			*) OCF_RESKEY_CRM_meta_timeout=100000 ;;
+		esac
+	fi
+	default_timeout=$(((${OCF_RESKEY_CRM_meta_timeout}/1000) - 5))
+	default_force_stop_timeout=$(((${OCF_RESKEY_CRM_meta_timeout}/1000) - 5))
+	: ${OCF_RESKEY_shutdown_timeout=${default_force_stop_timeout}}
+	: ${OCF_RESKEY_deep_probe_timeout=${default_timeout}}
+	: ${OCF_RESKEY_start_timeout=${default_timeout}}
+fi
+
+sybase_user_default="sybase"
+sybase_home_default="detect"
+ase_default="detect"
+ocs_default="detect"
+
+: ${OCF_RESKEY_sybase_user=${sybase_user_default}}
+: ${OCF_RESKEY_sybase_ase=${ase_default}}
+: ${OCF_RESKEY_sybase_ocs=${ocs_default}}
+: ${OCF_RESKEY_sybase_home=${sybase_home_default}}
+
+if [ "$__OCF_ACTION" != "meta-data" ]; then
+	if [ "$OCF_RESKEY_sybase_home" = "detect" ]; then
+		if [ -d "/opt/sap" ]; then
+			OCF_RESKEY_sybase_home="/opt/sap"
+		elif [ -d "/opt/sybase" ]; then
+			OCF_RESKEY_sybase_home="/opt/sybase"
+		else
+			ocf_log err "sybaseASE: Unable to detect 'sybase_home'."
+			exit $OCF_ERR_ARGS
+		fi
+	fi
+
+	sybase_env="$OCF_RESKEY_sybase_home/SYBASE.env"
+
+	if [ "$OCF_RESKEY_sybase_ase" = "detect" ]; then
+		if [ -f "$sybase_env" ]; then
+			OCF_RESKEY_sybase_ase=$(grep "SYBASE_ASE" "$sybase_env" | cut -d= -f2)
+		else
+			ocf_log err "sybaseASE: Unable to detect 'sybase_ase'."
+			exit $OCF_ERR_ARGS
+		fi
+	fi
+
+	if [ "$OCF_RESKEY_sybase_ocs" = "detect" ]; then
+		if [ -f "$sybase_env" ]; then
+			OCF_RESKEY_sybase_ocs=$(grep "SYBASE_OCS" "$sybase_env" | cut -d= -f2)
+		else
+			ocf_log err "sybaseASE: Unable to detect 'sybase_ocs'."
+			exit $OCF_ERR_ARGS
+		fi
+	fi
+fi
+
+
+interfaces_file_default="${OCF_RESKEY_sybase_home}/interfaces"
+: ${OCF_RESKEY_interfaces_file=${interfaces_file_default}}
+
+export LD_POINTER_GUARD=0
+
+#######################################################################################
+# Declare some variables we will use in the script. Please don't change their values. #
+#######################################################################################
+declare login_string=""
+declare RUNSERVER_SCRIPT=$OCF_RESKEY_sybase_home/$OCF_RESKEY_sybase_ase/install/RUN_$OCF_RESKEY_server_name
+declare CONSOLE_LOG=$OCF_RESKEY_sybase_home/$OCF_RESKEY_sybase_ase/install/$OCF_RESKEY_server_name.log
+
+##################################################################################################
+# This function will be called by rgmanager to get the meta data of resource agent "sybaseASE". #
+# NEVER CHANGE ANYTHING IN THIS FUNCTION.
+##################################################################################################
```

Comment:
How about dropping the "never change anything..." comments? :)

Reply:
Good idea. Wont be able to fix bugs if we cant change anything.